src/log.rs

```diff
@@ -22,6 +30,127 @@ impl Operation {
                 error,
                 latency: _,
             } => op.is_some() || error.is_some(),
+            Operation::CollectStats { per_dir_stats: _, loop_stats: _ } => {
```

Comment:
(I think) you can use the rest operator `{ .. }` here, rather than expanding each struct field and binding to `_`.

Reply:
```suggestion
            Operation::CollectStats { .. } => {
```